### DISCUSSION TITLE
Migration on Laravel Modules

### DISCUSSION THREAD

**Discussion by lukman-dev** [nWidart/laravel-modules#1236] [Q&A] [answered]
how to make migration with specific table ? 
I've tried it with the command 

`php artisan module:make-migration alter_table_product_prices_add_supplier_price --table=product_prices Product`
 OR
`php artisan module:make-migration alter_table_product_prices_add_supplier_price Product --table=product_prices `

but it's does not work

and how to rollback migration with step on laravel modules ?
I've tried it with the command 

`php artisan module:migrate-rollback --step=1 Product` OR `php artisan module:migrate-rollback Product --step=1`

it's also does not work

**Comment by SocolaDaiCa** [comment, marked as answer]
@lukman-dev try https://github.com/SocolaDaiCa/Laravel-Modules-Command

**Reply by lukman-dev** [reply]
this is very helpful, thank you very much

**Comment by khanfaisal9** [comment]
Look at the pictures,Google Images!
